src/interfaces.ts

```diff
@@ -115,6 +115,9 @@ export interface IPtyForkOptions {
   uid?: number;
   gid?: number;
   encoding?: string;
+  handleFlowControl?: boolean;
+  flowPause?: string;
```

Comment:
This probably doesn't need to be configurable? If people want it we could always add it at a later point if there's the need.

Reply:
Well if we want this to work with zsh where ppl tend to rebind XON/XOFF (those are the defaults) in xterm.js we already have that use case?